packages/cli/src/workflows/workflow-execution.service.ts

```diff
@@ -101,6 +101,12 @@ export class WorkflowExecutionService {
 		user: User,
 		pushRef?: string,
 	): Promise<{ executionId: string } | { waitingForWebhook: boolean }> {
+		// Store workflow active state before clearing it for manual execution
+		const workflowIsActive =
+			payload.workflowData.active ||
+			(typeof payload.workflowData.activeVersionId === 'string' &&
+				payload.workflowData.activeVersionId.length > 0);
```

Comment:
Technically this does not depend on the payload, but the status in the db. If user A has the workflow open and user B publishes it, then the payload and the db state will be out of sync and we will try to register another webhook.